ui/redux/selectors/publish.js

```diff
@@ -152,36 +157,40 @@ export const selectUploadCount = createSelector(
 export const selectIsScheduled = (state) => selectState(state).tags.some((t) => t.name === SCHEDULED_LIVESTREAM_TAG);
 
 // @flow
-export const selectCollectionClaimParamsForUri = (state: State, uri: string, collectionId: string) => {
-  const claim = selectClaimForUri(state, uri);
+export const selectCollectionClaimParamsForUri = (state: State, collectionId: string) => {
+  const claim = selectClaimForId(state, collectionId);
+
+  const uri = claim && claim.permanent_url;
   const collection = selectCollectionForId(state, collectionId);
   const activeChannelId = selectActiveChannelClaimId(state);
+  const hasEdits = selectCollectionHasEditsForId(state, collectionId);
+
+  if (activeChannelId === undefined) return undefined;
 
-  const { claim_id: claimId, name: collectionName, amount } = claim || collection || {};
+  const { name: collectionName } = collection || {};
 
   // Covers edited titles as well. It is separated out from collectionName as it
   // seems like collectionName wants to favor the claim's value for cases
   // outside of COL_TYPES.FEATURED_CHANNELS (not sure).
   const featuredTitle = collection?.type === COL_TYPES.FEATURED_CHANNELS ? collection.name : '';
 
-  const title = getClaimTitle(claim);
-  const collectionChannelId = getChannelIdFromClaim(claim);
+  const claimChannelId = getChannelIdFromClaim(claim);
   const tags = makeSelectMetadataItemForUri(uri, 'tags')(state);
   const languages = makeSelectMetadataItemForUri(uri, 'languages')(state);
   // removes falsey values from string array
   const collectionClaimIds = selectClaimIdsForCollectionId(state, collectionId).filter(Boolean);
 
   const collectionParams: CollectionPublishParams = {
-    thumbnail_url: collection ? collection.thumbnail?.url : getThumbnailFromClaim(claim),
-    name: collectionName ? sanitizeName(collectionName) : undefined,
-    description: collection ? collection.description : makeSelectMetadataItemForUri(uri, 'description')(state),
-    title: featuredTitle || (collection ? collectionName : title),
-    bid: String(amount || 0.001),
+    thumbnail_url: (hasEdits && collection.thumbnail?.url) || getThumbnailFromClaim(claim),
+    name: (hasEdits && sanitizeName(collectionName)) || undefined,
+    description: (hasEdits && collection.description) || makeSelectMetadataItemForUri(uri, 'description')(state),
+    title: featuredTitle || selectNameForCollectionId(state, collectionId),
+    bid: String(claim?.amount || 0.001),
     languages: languages ? dedupeLanguages(languages) : [],
     locations: makeSelectMetadataItemForUri(uri, 'locations')(state) || [],
     tags: tags ? tags.map((tag) => ({ name: tag })) : [],
-    claim_id: claimId,
-    channel_id: claim ? collectionChannelId : activeChannelId || undefined,
+    claim_id: claim && claim.claim_id,
+    channel_id: (claim ? claimChannelId : activeChannelId) || null,
     claims: collectionClaimIds,
   };
 
```

Comment:
:small_orange_diamond: Potential browser incompatibility warning

<img width="521" alt="image" src="https://user-images.githubusercontent.com/64950861/189810554-b3feadd2-6956-42cf-bf67-33bcb41549dd.png">


I think it's in `selectUpdatedAtForCollectionId`, roughly based on the stack trace.